airflow/www/views.py

```diff
@@ -2288,6 +2292,39 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, LoggingMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except Exception as e:
+            self.logger.error(e)
+            pass
+
+        # Get the Git repo and git hash
```

Comment:
@criccomini 
Can you remove this? It was a comment on the wrong line and though I have removed the git code, this comment persists.